default/scripting/monster_designs/SM_BLACK_KRAKEN.focs.txt

```diff
@@ -5,7 +5,7 @@ ShipDesign
     hull = "SH_BLACK_KRAKEN_BODY"
     parts = [
         "SR_SPINES"
-        "SR_SPINES"
+        "SR_TENTACLE"
```

Comment:
If you add one tentacle by a revision and do it bi-annual you'll get 8 tentacles in 4 years and Kraken will become an octopus :)